packages/extension-api-explorer/src/custom.d.ts

```diff
@@ -0,0 +1,4 @@
+declare module '*.svg' {
+  const content: any
+  export default content
+}
```

Comment:
Interesting we need this in both api-explorer and here

Reply:
Yeah. I alluded to this in the commit message. I tried various ways tsconfig options and putting the file in a single location but did not have any luck. This is the only way I got the tsc command to work. babel does not have any issues but we are still using tsc for emitting the type declarations